clang/include/clang/Basic/TargetInfo.h

```diff
@@ -220,6 +220,15 @@ enum OpenCLTypeKind : uint8_t {
   OCLTK_Sampler,
 };
 
+// ISO/IEC TS 18661-2, ISO/IEC TR 24733, and C23 decimal floating-point.
+// Multiple encodings for how Decimal floating-point values are stored are
+// used in practice. This enumeration provides an enumerator for each known
```

Comment:
Fix comment here.

Reply:
I see the "Decimal" capitalization error. Was there anything else that needs fixing?

Reply:
The 2 "are" ?

Reply:
Ah, I think that is grammatically ok, but it does sound awkward. Is this better?
```
// ISO/IEC TS 18661-2, ISO/IEC TR 24733, and C23 decimal floating-point.
// Multiple encoding forms for the storage of decimal floating-point values
// have been defined for use on various platforms. This enumeration provides
// an enumerator for each known encoding.
```

Reply:
Perfect!